lib/topological_inventory/ansible_tower/parser/service_offering.rb

```diff
@@ -7,21 +7,14 @@ def parse_service_offering(template_hash)
         service_inventory = lazy_find(:service_inventories, :source_ref => template.inventory_id.to_s) if template.respond_to?(:inventory_id)
 
         extra = {
-          :type                    => template_hash[:template_type],
-          :ask_variables_on_launch => template.ask_variables_on_launch,
+          :type                    => template.type.to_sym,
           :ask_inventory_on_launch => template.ask_inventory_on_launch,
           :survey_enabled          => template.survey_enabled,
         }
 
         if template_hash[:template_type].to_s == 'job_template'
           extra = extra.merge(
             :ask_credential_on_launch => template.ask_credential_on_launch,
-            :ask_diff_mode_on_launch  => template.ask_diff_mode_on_launch,
```

Comment:
@mkanoor are you using any of removed values in catalog?

Reply:
@slemrmartin We are not using any of those fields.

```
                "ask_credential_on_launch",
                "ask_tags_on_launch",
                "ask_diff_mode_on_launch",
                "ask_skip_tags_on_launch",
                "ask_job_type_on_launch",
                "ask_limit_on_launch",
                "ask_verbosity_on_launch"
```

Other than survey_enabled I don't think there is anything else we need

Reply:
@mkanoor Thanks! I'll keep `ask_inventory_on_launch` and `ask_credential_on_launch` because they're related to input forms which should be opened in Catalog before Template is ordered (if Template requires it)

It's not related to this PR, but if Catalog is not supporting credentials and `ask_credential_on_launch`, only subset of templates can be ordered.